sensor/docs/tutorials/tag_based_sfm_calibrator.md

```diff
@@ -0,0 +1,360 @@
+  </tr>
+  <tr>
+    <td><img src="../images/tag_based_sfm_calibrator/intrinsics_good_3.jpg" alt="intrinsics_3" width = 400px ></td>
+    <td><img src="../images/tag_based_sfm_calibrator/intrinsics_good_4.jpg" alt="intrinsics_4" width = 400px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Good example. Focuses on the corners of the image</p></td>
+    <td><p style="text-align: center;">Good example. Adds variation in the distance from the camera</p></td>
+  </tr>
+  <tr>
+    <td><img src="../images/tag_based_sfm_calibrator/intrinsics_bad_1.jpg" alt="intrinsics_5" width = 400px ></td>
+    <td><img src="../images/tag_based_sfm_calibrator/intrinsics_bad_2.jpg" alt="intrinsics_6" width = 400px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Bad example. The board is not completely covered in the image</p></td>
+    <td><p style="text-align: center;">Bad example. The board is not completely covered in the image</p></td>
+  </tr>
+</table>
+
+Once enough images have been collected, transfer the images from the camera to the computer, and press `Calibrate external camera intrinsics` as shown in the following image:
+
+<p align="center">
+  <img src="../images/tag_based_sfm_calibrator/menu1.png" alt="menu1"/>
+</p>
+
+If there are no issues, the output in the console should be similar to the following one:
+
+```text
+[tag_based_sfm_calibrator-1] [INFO] [1713488090.641210235] [intrinsics_calibrator]: Camera matrix:
+[tag_based_sfm_calibrator-1] [4816.165518235453, 0, 2160.348608760617;
+[tag_based_sfm_calibrator-1]  0, 4821.30868416463, 1552.554137385826;
+[tag_based_sfm_calibrator-1]  0, 0, 1]
+[tag_based_sfm_calibrator-1] [INFO] [1713488090.641218916] [intrinsics_calibrator]: Distortion coefficients:
+[tag_based_sfm_calibrator-1] [-0.07340778013968814, 0.02950049592323137, 0.005167166916447808, 0.002758631510330731, 0]
+[tag_based_sfm_calibrator-1] [INFO] [1713488090.641265465] [intrinsics_calibrator]: Undistorted camera matrix:
+[tag_based_sfm_calibrator-1] [4750.68994140625, 0, 2168.846093300279;
+[tag_based_sfm_calibrator-1]  0, 4784.35107421875, 1560.251275314979;
+[tag_based_sfm_calibrator-1]  0, 0, 1]
+[tag_based_sfm_calibrator-1] [INFO] [1713488092.917012306] [tag_based_sfm_calibrator]: k = [4816.165518235453, 0, 2160.348608760617;
+[tag_based_sfm_calibrator-1]  0, 4821.30868416463, 1552.554137385826;
+[tag_based_sfm_calibrator-1]  0, 0, 1]
+[tag_based_sfm_calibrator-1] [INFO] [1713488092.917062856] [tag_based_sfm_calibrator]: d = [-0.07340778013968814, 0.02950049592323137, 0.005167166916447808, 0.002758631510330731, 0]
+[tag_based_sfm_calibrator-1] [INFO] [1713488092.917079752] [tag_based_sfm_calibrator]: new_k = [4750.68994140625, 0, 2168.846093300279;
+[tag_based_sfm_calibrator-1]  0, 4784.35107421875, 1560.251275314979;
+[tag_based_sfm_calibrator-1]  0, 0, 1]
+```
+
+## Scene data collection
+
+To reconstruct the scene and find the `base_link`` pose, we need to obtain data from one or more scenes.
```

Comment:
`base_link`` to `base_link`

Reply:
Thanks. Addressed in [877a2f7](https://github.com/tier4/CalibrationTools/pull/164/commits/877a2f7dc110a733be7e9043595f6e57922ff025)